### PR TITLE
doc: add more details and examples to import openapi guide

### LINKED ISSUE #1464: doc: improve documentation for route export and import
# Feature request

## Please describe your feature

We need to describe the route import and export feature in detail, including the processing details of upstream, service, plugins and other data in route, and other details.

There are most of the descriptions in the relevant proposals, we could refer to them.

### LINKED ISSUE #1660: doc: Improve docs about route import and export
Our current doc about route import is too simple, see:
https://github.com/apache/apisix-dashboard/blob/master/docs/en/latest/IMPORT_OPENAPI_USER_GUIDE.md

We need to supplement doc about route export.
The docs should be described in more detail.
Such as:
1. What does the imported content include?
2. How to deal with duplicate imported routes
3. Import standard swagger documents, how to deal with data that apisix does not have
4. What data to export
5. When exporting, apisix has it, but what will happen to the data that is not in the swagger standard?
etc..

### PR DESCRIPTION
**Pull request by liuxiran**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

improve `IMPORT_OPENAPI_USER_GUIDE.md`, add more details and examples to help users to import route from the correct OAS3.0 file.

**Related issues**

close #1464 
close #1660 

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [x] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Review by nic-chen — COMMENTED** [review]
@liuxiran 

thanks so much.

Need to add some instructions on importing duplicate data, as well as instructions on the `status` of the route after import.

File: docs/en/latest/IMPORT_OPENAPI_USER_GUIDE.md [hunk 1]
```diff
@@ -21,34 +21,21 @@ title: Import OpenAPI Guide
...
+Apache APISIX Dashboard supports importing [OpenApi3.0](https://swagger.io/specification/)(we will use OAS3.0 for short) files, both `json` and `yaml` are supported, to create one or more Routes. Currently we support most of the OpenApi specifications, but there are some differences, which are in terms of compatibility and extended fields.
 
 ## Extended fields
 
-There are some fields required in APISIX Route but are not included in the properties of OAS3.0, we added some extended fields such as upstream, plugins, hosts and so on. All extensions start with x-apisix. See [reference](https://apisix.apache.org/docs/apisix/admin-api/#route) For more details of the APISIX Route Properties
+There are some fields required in APISIX Route but are not included in the properties of OAS3.0, inorder to provide convenience for extending custom route entities based on OAS3.0, we added some extended fields such as upstream, plugins, hosts and so on. All extensions start with x-apisix. See [reference](https://apisix.apache.org/docs/apisix/admin-api/#route) For more details of the APISIX Route Properties
```
Comment (batman-ezio) [inline docs/en/latest/IMPORT_OPENAPI_USER_GUIDE.md:32]: inorder to `in order to `

**Comment by starsz** [comment]
@juzhiyuan @LiteSun Help check the Frontend CI.

**Comment by juzhiyuan** [comment]
image

Hi, please first update the markdown

File: docs/en/latest/IMPORT_OPENAPI_USER_GUIDE.md (hunk 1 above)
Comment (liuxiran) [reply docs/en/latest/IMPORT_OPENAPI_USER_GUIDE.md]: fixed, thanks

**Review by nic-chen — APPROVED** [review]
LGTM from the content. Thanks @liuxiran

**Review by LiteSun — APPROVED** [review]

File: docs/en/latest/IMPORT_OPENAPI_USER_GUIDE.md [hunk 2]
```diff
@@ -58,5 +45,407 @@ There are some fields required in APISIX Route but are not included in the prope
...
+        '200':
+          description: list response
+        default:
+          description: unexpected error
+...
+```
+
+## Examples about how to config the OAS3.0 in different user scenarios
+
+### config a basic published route
```
Comment (juzhiyuan) [inline docs/en/latest/IMPORT_OPENAPI_USER_GUIDE.md:160]: actually, `config` is not a variable.

Use `Configure` here.

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Review by LiteSun — APPROVED** [review]

**Review by starsz — APPROVED** [review]

**Review by imjoey — APPROVED** [review]
/lgtm

**Pull request merged**